editor/editor_property_name_processor.cpp

```diff
@@ -237,6 +242,7 @@ EditorPropertyNameProcessor::EditorPropertyNameProcessor() {
 	capitalize_string_remaps["webp"] = "WebP";
 	capitalize_string_remaps["webrtc"] = "WebRTC";
 	capitalize_string_remaps["websocket"] = "WebSocket";
+	capitalize_string_remaps["wine"] = "wine";
```

Comment:
The official way to capitalize [Wine](https://www.winehq.org/) is "Wine", why was this changed?